compose-setup.sh

```diff
@@ -3,24 +3,10 @@
 set -e
 
 check_version() {
-	VER=$(docker-compose version --short)
-	if [ "$VER" = "1.5.1" ]; then
-		# Known to have bugs when generating configs.
-cat >&2 <<EOF
-Your docker-compose version is known to have bugs that make Portus unusable. The
-known affected version(s) are:
-  - 1.5.1
-Updating or downgrading to any other version will solve this issue. The file
-docker/registry/config.yml will also be deleted (as any configuration files
-generated by the above versions of docker-compose need to be replaced).
-EOF
-		if [ "$FORCE" -ne 1 ]; then
-			rm -f "docker/registry/config.yml"
-			exit 1
-		else
-			echo "Ignoring docker-compose version since -f was given." >&2
-		fi
-	fi
+    if [ "$VER" = $(echo -e '$VER\n1.6' | sort -V | head -n1) ]; then
```

Comment:
The variable `VER` is not set anywhere. I guess we should not remove the line `VER=$(docker-compose version --short)` :wink: